core/templates/dev/head/components/ck-editor-helpers/ck-editor-widgets.initializer.ts

```diff
@@ -77,6 +78,24 @@ oppia.run([
                 customizationArgSpecs,
                 customizationArgs,
                 function(customizationArgsDict) {
+                  // Adding imageId and removing filepath.
+                  if (ckName === 'oppiaimage') {
```

Comment:
oppiaimage should be a named constant somewhere I think. If so use that.

Reply:
Missed this comment?

Reply:
sorry, 
done